.github/workflows/unittest-linux-cpu.yml

```diff
@@ -1,69 +1,72 @@
 name: Unit-tests on Linux CPU
 
 on:
-  pull_request:
-  push:
-    branches:
-      - nightly
-      - main
-      - release/*
-  workflow_dispatch:
+    pull_request:
```

Comment:
If possible, revert to the previous indentation, because it makes it hard to know what load-bearing changes were made in the PR and thus complicates the review